src/security/store.js

```diff
@@ -60,6 +61,7 @@ const actions = {
       })
         .then(data => {
           localStorage.setItem("token", data.access_token);
+          localStorage.setItem("token_expire", getExpireDate(data.expires_in));
```

Comment:
It is hard to say what `token_expire` is. I think `token_expires_at` would be better and easy to understand it is a date.